.vsts-ci.yml

```diff
@@ -0,0 +1,80 @@
+trigger:
+- master
+
+resources:
+  repositories:
+  # shared library repository
+  - repository: arcade
+    type: github
+    endpoint: DotNet-Bot GitHub Connection
+    name: dotnet/arcade
+    ref: refs/heads/master
+
+variables:
+  teamName: Roslyn-Project-System
+  ${{ if eq(variables['Build.Reason'], 'PullRequest') }}:
+    PB_PublishBlobFeedKey:
+    PB_PublishBlobFeedUrl:
+  ${{ if ne(variables['Build.Reason'], 'PullRequest') }}:
+    PB_PublishBlobFeedUrl: https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json
+
+phases:
+- template: /eng/build.yml
+  parameters:
+    agentOs: Windows_NT
+    queue:
+      ${{ if eq(variables['Build.Reason'], 'PullRequest') }}:
+        name: Hosted VS2017
+      ${{ if ne(variables['Build.Reason'], 'PullRequest') }}:
+        name: DotNetCore-Build
+        demands:
+        - agent.os -equals Windows_NT        
+      parallel: 2
+      matrix:
+        ${{ if eq(variables['Build.Reason'], 'PullRequest') }}:
+          Build_Debug:
+            _BuildConfig: Debug
+            _PublishType: none
+            _SignType: test
+        Build_Release:
+          _BuildConfig: Release
+          ${{ if eq(variables['Build.Reason'], 'PullRequest') }}:
+            _PublishType: none
+            _SignType: test
+          ${{ if ne(variables['Build.Reason'], 'PullRequest') }}:
+            _PublishType: blob
+            _SignType: real
+
+- ${{ if eq(variables['Build.Reason'], 'PullRequest') }}:
+  - template: /eng/build.yml
+    parameters:
+      agentOs: Linux
+      dockerImage: microsoft/dotnet-buildtools-prereqs:ubuntu-14.04-cross-0cd4667-20170319080304
```

Comment:
This line isn't actually doing anything anymore.  I'm fixing this in the CliCommandLineParser repo with my current changes that will enable telemetry, but that's not quite ready yet.  The way to do this is the way it's implemented in Arcade today (this changed recently, so it makes sense that you followed the old model).  Delete this line and then add a variables entry like https://github.com/dotnet/arcade/blob/master/.vsts-ci.yml#L51-L52